test/data/config.json

```diff
@@ -7,6 +7,7 @@
 	"fileLogLevel": "error",
 	"logFileName": "logs/lisk.log",
 	"consoleLogLevel": "none",
+	"wsEngine": "sc-uws",
```

Comment:
Whenever make any changes in configuration file, please update `schema/config.js` respectively.